### DISCUSSION TITLE
Negative Transaction Fee

### DISCUSSION THREAD

**Discussion by SaftCPU** [lightningnetwork/lnd#8480] [Ideas]
I have some channels that don't route even with zero fees. My partner has astronomically high fees. Profitable rebalancing is therefore not possible. What if you could pay others to use the channel? (neative fees)

**Comment by guggero** [comment]
You mean "inbound fees"?
See https://github.com/lightning/blips/pull/18 and https://github.com/lightningnetwork/lnd/pull/6703.

**Comment by SaftCPU** [comment]
No, I mean paying someone for using the Channel. Giving sats for routing instead of charging a fee.

**Reply by guggero** [reply]
Ah, right. So https://github.com/lightning/bolts/pull/1022.
